frontend/src/metabase/visualizations/visualizations/SmartScalar/SettingsComponents/PeriodsAgoMenuOption.tsx

```diff
@@ -2,14 +2,15 @@ import type { KeyboardEvent, MouseEvent } from "react";
 import { useCallback, useRef, useState } from "react";
 import { t } from "ttag";
 
-import type { NumberValue } from "metabase/querying/filters/hooks/use-number-filter";
 import { Box, Group, Text, rem } from "metabase/ui";
 import type { COMPARISON_TYPES } from "metabase/visualizations/visualizations/SmartScalar/constants";
 import type { SmartScalarComparisonPeriodsAgo } from "metabase-types/api";
 
 import { MenuItemStyled } from "./MenuItem.styled";
 import { NumberInputStyled } from "./PeriodsAgoMenuOption.styled";
 
+type NumberValue = number | "";
```

Comment:
Use the same type as beforehttps://github.com/metabase/metabase/blob/2df4df7a8c1b43504109c668fb9d9ac6c6c92488/frontend/src/metabase/querying/filters/hooks/use-number-filter/types.ts#L10